articles/storage/files/storage-files-planning.md

```diff
@@ -78,7 +78,8 @@ Standard file shares are backed by hard disk drives (HDDs). Standard file shares
 Standard file shares up to 5 TiB in size are available as a GA offering. While larger file shares, which are any shares larger than 5 TiB, up to a maximum of 100 TiB, are currently available as a preview offering.
 
 > [!IMPORTANT]
-> - Requires you to create a new general purpose storage account (cannot expand existing storage accounts).
+> Large (> 5TiB) File shares:

```

Comment:
Thanks for making this change, we appreciate it! We're trying to be selective with our wording here though so could you alter this to: "File shares larger than 5 TiB:"

If you do that, I'd be happy to sign off.

Thanks again!